PowerEditor/src/WinControls/TaskList/TaskListDlg.cpp

```diff
@@ -38,16 +38,16 @@ LRESULT CALLBACK hookProc(UINT nCode, WPARAM wParam, LPARAM lParam)
 	if ((nCode >= 0) && (wParam == WM_RBUTTONUP))
     {
 		::PostMessage(hWndServer, WM_RBUTTONUP, 0, 0);
-    }        
-	if ((nCode >= 0) && (wParam == WM_MOUSEWHEEL))
+    } 
+	if ((nCode >= 0) && (wParam == WM_MOUSEWHEEL) && windowsVersion== WV_WIN10)

```

Comment:
In my Win7 and Win8.1 tests the mouse wheel messages are sent to the control even if the mouse is outside the control whereas on Win10 they stop if you move outside the task list. So it appears to that Win10 handles things slightly differently, and that's what I put the check to only pass the messages on if its on Win10.